pkg/cmd/pulumi/org.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/pulumi/pulumi/pkg/v3/backend/display"
+	"github.com/pulumi/pulumi/pkg/v3/backend/httpstate"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
+)
+
+func newOrgCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "org",
+		Short: "Manage Organisation configuration",
+		Long: "Manage Organisation configuration.\n" +
+			"\n" +
+			"Use this command to manage any specific organization configuration, " +
+			"e.g. setting the default organization for a backend",
+		Args: cmdutil.NoArgs,
+	}
+
+	cmd.AddCommand(newOrgSetCmd())
+	cmd.AddCommand(newOrgGetCmd())
+
+	return cmd
+}
+
+func newOrgSetCmd() *cobra.Command {
+	var orgName string
+
+	var cmd = &cobra.Command{
+		Use:   "set [NAME]",
+		Args:  cmdutil.MaximumNArgs(1),
+		Short: "Set the default org for the current backend",
```

Comment:
We mix "org" and "organization" - in plain english, should we consistently use "organization"?